### ISSUE TITLE
Een belangrijk bericht

### ISSUE THREAD

**Issue by Z3r0byte** [Z3r0byte/Magis#77]
Ik stop met het verder ontwikkelen van Magis. Magis gaat al een tijd constant achteruit met het aantal installaties en dit is begrijpelijk, want Magis biedt nou eenmaal niet zoveel functies als sommige andere Magister apps. Daarnaast heeft Magister afgelopen jaar een nieuwe manier van inloggen ingevoerd en het is maar de vraag hoe lang de oude manier nog zal blijven werken. Ik heb er simpelweg de tijd niet voor om naast school en werk (apps maken maakt je helaas niet rijk 😉) nog mijn apps te onderhouden en te updaten met nieuwe functies, zeker niet nu ik in mijn examenjaar zit. Daarom heb ik besloten te stoppen met het verder ontwikkelen van Magis.

**Comment by UnderKoen** [comment]
Dat is vandaag unsupported geworden.

**Comment by Z3r0byte** [comment]
Ja, ik merkte al zoiets. Ik zal binnenkort de app uit de PlayStore halen om verwarring te voorkomen.

**Comment by UnderKoen** [comment]
Ik heb al een manier gevonden om een bearer token te krijgen. Ga binnen kort implementatie schrijven voor swift.

**Comment by Z3r0byte** [comment]
Nice, succes!
